int8 INACTIVE=0
int8 ACTIVE=1

std_msgs/Header header
int8 state
